task1/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="Author" content="Andrey T."> 
+    <title>Задание 1</title>
+    <link href="css.css" rel="stylesheet">
```

Comment:
Change CSS file name. Example main.css, index.css. CSS file name should be similar with HTML name